components/menu/mobile.css

```diff
@@ -50,6 +50,26 @@
   /* Title */
 
   .menu__panel-title {
-    border-bottom: 1px solid var(--color-border-primary);
+    &::after {
+      position: relative;
+      top: 1rem;
+
+      display: block;
+
+      height: 1px;
+
+      content: "";
+
+      background-color: var(--color-border-primary);
+    }
+  }
```

Comment:
Why does `border-bottom` not work here?

Reply:
<img width="378" height="85" alt="image" src="https://github.com/user-attachments/assets/f63829d0-75f8-4a1e-86e0-9ca6792140f5" />

It would be full-width